gke-deploy/doc/gke-deploy_prepare.md

```diff
@@ -32,6 +32,7 @@ gke-deploy prepare [flags]
 
 ```
   -a, --app string         Application name of the Kubernetes deployment.
+  -x, --expose int         Creates a Service resource that connects to a deployed resource using a selector that matches the value provided by --app. The port provided will be used to expose the deployed resource (i.e., port and targetPort will be set to the value provided in this flag).
   -f, --filename string    Config file or directory of config files to use to create the Kubernetes resources (file or files in directory must end with ".yml" or ".yaml").
   -h, --help               help for prepare
   -i, --image strings      Image(s) to be deployed. Images can be set comma-delimited or as separate flags.
```

Comment:
Unrelated to this PR but the comment for `-namespace` should be "Namespace of GKE cluster to deploy to"